web/src/Components/Shared/Forms/index.tsx

```diff
@@ -1,7 +1,4 @@
 import Text from './Text';
 import Password from './Password';
 
-export {
-  Text,
-  Password,
-}
\ No newline at end of file
+export { Text, Password };
```

Comment:
Nit: there is a syntactic sugar for this
```suggestion
export { default as Text } from './Text';
export { default as Password } from './Password';
```

Reply:
very cool.  love it.